### ISSUE TITLE
508-defect-2: Headings should be consistent and logically ordered

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#54724]
## Point of contact

joshkimux

## Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

## Details

Currently, when inspecting the page's headings, two issues emerge
1. Records and document titles are not headings, despite starting subsections
2. There is no programmatic and (clear) visual delineation between "add more evidence" and the section directly above it @eileen-coforma 

<img width="1313" alt="Screenshot 2023-03-07 at 1 17 18 PM" src="https://user-images.githubusercontent.com/14154792/223513854-912b1103-cd0d-4336-965c-0debb521594a.png">

## Reproduction steps

1. Add multiple facilities and pieces of evidence
2. [On the review page](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/supporting-evidence/summary), inspect the titles, and confirm they are not headings
3. Confirm that "add new evidence"'s immediate heading is the `h4` You uploaded these documents, implying it is a part of the documents section (and not applicable to the broader page)

### Titles should be `h5`s
Document and facility titles should be `h5`s as they begin sections with actions 

<img width="360" alt="Screenshot 2023-03-07 at 1 21 28 PM" src="https://user-images.githubusercontent.com/14154792/223514506-765f6d7f-bad3-4842-8fa5-ba97697cdd62.png">

### Design to review and determine a path forwards for add more evidence
Add more evidence may require more design work to distinguish it _visually_ as separate to _only_ documents, and applicable to the full page. This may then impact how we _code_ the add more evidence link to be separate as well from a programmatic standpoint.

#### Success criteria
- [x] It is visually clear that "Add more evidence" is not specifically grouped to documents
- [x] It is programmatically clear that "Add more evidence" is not a sub-section to documents
  - [x] Add a `h4` "Are you missing evidence?" above the "Add more evidence" link
- [x] Solution is consistent with the contestable-issues page - "supporting-evidence/summary" page has bolded titles as heading level 5
- [x] Solution is consistent with the review-and-submit page - has bolded titles as heading level 6

## References, articles, or WCAG support

1.
2.
3.
...

## Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [X] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by Mottie** [comment]
Note: the `h5`s may need to be updated to be `h6`s on the review & submit page... I can't say it's a definite thing until it's looked at.

**Comment by joshkimux** [comment]
@Mottie Let me give that a look. I wonder if it may be easier to flag the entire review page as a defect so our team isn't responsible for fixing what is a super wicked problem  😓 (given how rough it is on how the heading levels pan out)

**Comment by Mottie** [comment]
We can control the heading levels on the review and submit page... I wanted to add a note to share that we need to be aware of this issue.

**Comment by joshkimux** [comment]
That's a relief to hear! I can add it to the SC!

**Comment by joshkimux** [comment]
@Mottie I talked to @eileen-coforma , and we wriggled around this for a bit and realized it's a pretty wicked problem that needs a little more love. But in the meantime, to reduce this from a `defect-2` we agreed to try a `sr-only` heading to split it apart for the screen reader experience.

<img width="1276" alt="Screenshot 2023-03-10 at 2 37 19 PM" src="https://user-images.githubusercontent.com/14154792/224412038-db620208-3ec7-426b-8854-e072480f1918.png">

**Comment by rocio-desantiago** [comment]
@joshkimux Is the h4 class supposed to say `Are you missing evidence`? 
I'm trying to search for that. :D

**Comment by joshkimux** [comment]
@Mottie yes! Bingo. It is one of the checkboxes in the original ticket.

**Comment by rocio-desantiago** [comment]
@joshkimux ah that makes sense

**Comment by Mottie** [comment]
LOL, @joshkimux my avatar is completely different from Rocio's 🙈

**Comment by joshkimux** [comment]
@rocio-desantiago oh my god I’m so sorry, was writing this at the doctors office on painkillers on my phone 🙇‍♂️🙇‍♂️🙇‍♂️🙇‍♂️ will wait until the next day next time

**Comment by rocio-desantiago** [comment]
@eileen-coforma any feedback on how to make "Add more evidence" visually clear and that it's not specifically grouped to documents?

**Comment by joshkimux** [comment]
@rocio-desantiago this is a _really_ good catch! In this case, "do you want to limit..." would be the `h5` and "Yes I want to limit..." would be a regular paragraph.

**Comment by rocio-desantiago** [comment]
@joshkimux For clarification: Do we want to unbold the "Yes, I want to limit..."
Otherwise, it'll look like this: 
![questionAsH5](https://user-images.githubusercontent.com/37054305/227655724-b32cd522-9691-41a6-82fb-b3014affefc9.png)

**Comment by eileen-coforma** [comment]
@rocio-desantiago I think 'Do you want to limit the information' would be H6, since it's tied to the section above with the H5 'We're requesting records from these private medical records'. 

The 'Yes, I want to limit the information requested' would not need be bolded.

CC: @joshkimux

**Comment by joshkimux** [comment]
@rocio-desantiago will check this first thing tomorrow to validate when it's up on staging ❤️

**Comment by rocio-desantiago** [comment]
@eileen-coforma fixed 👍

**Comment by joshkimux** [comment]
<img width="393" alt="Screenshot 2023-03-29 at 4 56 44 PM" src="https://user-images.githubusercontent.com/14154792/228665563-312974d0-b463-41e1-9b3f-a598b9679701.png">

Confirming this has been fixed 😄
